core/src/main/scala/coulomb/ops/standard/pow.scala

```diff
@@ -26,18 +26,19 @@ import coulomb.ops.{Pow, SimplifiedUnit}
 import coulomb.rational.typeexpr
 import coulomb.ops.algebra.FractionalPower
 
-transparent inline given ctx_pow_FractionalPower[V, U, E](using
+given ctx_pow_FractionalPower[V, U, E](using
     alg: FractionalPower[V],
+    dbv: typeexpr.DoubleValue[E],
     su: SimplifiedUnit[U ^ E]
         ): Pow[V, U, E] =
-    val e = typeexpr.double[E]
+    val e = dbv.value
     new Pow[V, U, E]:
```

Comment:
Question: instead of making this an anonymous class within the `inline` function, can we make it a named class outside that is then instantiated within the `inline` function? The problem with anonymous classes in `inline` functions it that means everywhere the function is `inline`d, you will get a completely new (anonymous) class. This causes code bloat, esp. on JS.

Reply:
See #265. If I'm understanding what you are thinking, it looks promising